README.md

```diff
@@ -18,7 +18,7 @@ Include in your project as a Maven dependency:
 <dependency>
     <groupId>net.quedex</groupId>
     <artifactId>java-api</artifactId>
-    <version>0.7.0</version>
+    <version>0.8.0</version>
```

Comment:
I'd first publish, then change version here.

Reply:
Shouldn't it be 0.8.0-SNAPSHOT for now?